tests/test_integration.py

```diff
@@ -674,15 +670,11 @@ async def aggregate_and_redeem_tickets():
                 redeemed_value = balance_str_to_int(statistics_after.redeemed_value) - balance_str_to_int(
                     statistics_before.redeemed_value
                 )
-                redeemed_ticket_count = statistics_after.redeemed - statistics_before.redeemed
 
                 if redeemed_value >= AGGREGATED_TICKET_PRICE:
                     break
                 else:
-                    await asyncio.sleep(0.5)
-
-            assert redeemed_value >= AGGREGATED_TICKET_PRICE
-            assert redeemed_ticket_count == pytest.approx(redeemed_value / AGGREGATED_TICKET_PRICE, 0.1)
```

Comment:
How do we now check that the aggregation worked? Didn't we check previously that the count of unredeemed tickets reduced from N to 1?

Reply:
Indeed. With the deprecation of listing the tickets per channel, this is no longer possible. But then the API functionality seems to have been there exclusively for testing.

We could observe the external behavior, the assumption is that:
1. the aggregated value equals that before aggregation
2. there's a single redeem operation

2nd of which can effectively not be done from the API now.

You know that aggregation passed, because the aggregate request returned 200.